eng/emitter-package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "main": "dist/src/index.js",
+  "dependencies": {
+    "@azure-tools/typespec-go": "https://github.com/Azure/autorest.go/releases/download/v0.1.0-dev.7-20240701.1/azure-tools-typespec-go-0.1.0-dev.7-20240701.1.tgz"
```

Comment:
Let's wait then update to the next version which Joel is working on.

Reply:
used typespec-go@v0.1.0